### PR TITLE
Create new Releasing with an advisory doc

### PR DESCRIPTION
**Pull request by mmalina**
Initial doc for advisories.

### CODE REVIEW

File: docs/modules/ROOT/pages/how-to-guides/proc_release_advisory.adoc [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+   advisory: <advisory data> <.>
+----
+
++
+<.> Advisory data to be included.
+
+.*UMB messages*
+
+When an advisory is created, a message is sent automatically to UMB (Universal Message Bus).
+The message will be sent to topic VirtualTopic.eng.konflux.advisory .
```
Comment (johnbieren) [inline docs/modules/ROOT/pages/how-to-guides/proc_release_advisory.adoc:34]: Space before period probably isn't needed

File: docs/modules/ROOT/pages/how-to-guides/proc_release_advisory.adoc [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ data:
+   advisory: <advisory data> <.>
+----
+
++
+<.> Advisory data to be included.
+
+.*UMB messages*
+
+When an advisory is created, a message is sent automatically to UMB (Universal Message Bus).
```
[the remark this answers is not in the document]
Comment (mmalina) [reply docs/modules/ROOT/pages/how-to-guides/proc_release_advisory.adoc]: Updated.

File: docs/modules/ROOT/pages/how-to-guides/proc_release_advisory.adoc (hunk 1 above)
Comment (mmalina) [reply docs/modules/ROOT/pages/how-to-guides/proc_release_advisory.adoc]: OK, removed.

File: docs/modules/ROOT/pages/how-to-guides/proc_release_advisory.adoc [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
++
+[source,yaml]
+----
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleasePlanAdmission
+metadata:
+  ..
+spec:
+ data:
+   advisory: <advisory data> <.>
```
[the remark this answers is not in the document]
Comment (mmalina) [reply docs/modules/ROOT/pages/how-to-guides/proc_release_advisory.adoc]: ok, updated, but I still kept `<advisory data>`. Please let me know if that should go as well.

File: docs/modules/ROOT/pages/how-to-guides/proc_release_advisory.adoc
[the remark this answers is not in the document]
Comment (mmalina) [reply docs/modules/ROOT/pages/how-to-guides/proc_release_advisory.adoc:1]: OK, added.

**Review by johnbieren — APPROVED** [review]

**Review by scoheb — APPROVED** [review]

**Review by ralphbean — APPROVED** [review]

File: docs/modules/ROOT/pages/how-to-guides/proc_release_advisory.adoc [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+[source,yaml]
+----
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleasePlanAdmission
+metadata:
+  ..
+spec:
+ data:
+   releaseNotes: <advisory data> <.>
```
Comment (ralphbean) [inline docs/modules/ROOT/pages/how-to-guides/proc_release_advisory.adoc:23]: There's not much information here for a user to go on. We should flesh this out more in a followup PR.

File: docs/modules/ROOT/pages/how-to-guides/proc_release_advisory.adoc [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+----
+<.> Advisory data to be included.
+
+.*UMB messages*
+
+When an advisory yaml is created and pushed to the advisories Gitlab repo, a message is sent automatically to UMB (Universal Message Bus) via Gitlab CI.
```
Comment (ralphbean) [inline docs/modules/ROOT/pages/how-to-guides/proc_release_advisory.adoc:29]: Also, this doesn't make sense in the upstream docs. We can leave it here for now, but we should move this out to a downstream-only doc once we have a process established to integrate those.

Comment (johnbieren) [reply docs/modules/ROOT/pages/how-to-guides/proc_release_advisory.adoc:29]: Should we just move the entire doc to a downstream doc? I am not sure it makes sense to have any of it upstream. Maybe the advisory repo README?

Comment (mmalina) [reply docs/modules/ROOT/pages/how-to-guides/proc_release_advisory.adoc:29]: I agree the whole doc would probably make sense somewhere internally, but not sure where.

**Pull request merged**